cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillLauncher.java

```diff
@@ -157,4 +166,70 @@ private void deletePod(PodInfo podInfo) {
       LOG.warn("Failed to delete pod {} with uid {}", podInfo.getName(), podInfo.getUid(), e);
     }
   }
+
+  /**
+   * Get instance id from the annotation. The annotation will have array of boolean
+   */
+  private InstanceInfo getInstanceInfo(PodInfo podInfo) throws Exception {
```

Comment:
nit: 'get' usually indicates only a read is happening. A name like 'claimInstanceId' or something like that would be more descriptive.